x-pack/plugin/ilm/qa/multi-cluster/src/test/java/org/elasticsearch/xpack/ilm/CCRIndexLifecycleIT.java

```diff
@@ -341,7 +342,7 @@ public void testAliasReplicatedOnShrink() throws Exception {
             assertBusy(() -> assertOK(client().performRequest(new Request("HEAD", "/" + shrunkenIndexName + "/_alias/" + indexName))));
 
             // Wait for the index to complete its policy
-            assertBusy(() -> assertILMPolicy(client(), shrunkenIndexName, policyName, "completed", "completed", "completed"));
+            assertBusy(() -> assertILMPolicy(client(), shrunkenIndexName, policyName, null, "complete", "complete"));
```

Comment:
I'm probably missing something, but why is the phase `null` in these CCR cases? Should this be the name of the phase that was completed? (ie. warm)

Reply:
`null` just means that the matcher should not try to match the phase. The reason for that is that in these tests there is a `randomBoolean()` check that randomly uses either a lone `warm` phase or adds a `cold` phase, so we don't assert whether it's always `cold` or `warm` (it's only important that it reaches the `complete` step)

Reply:
Ah gotcha, the implementation skips that assertion if the phase is `null`. Thanks for the clarification, this makes sense